tools/isolate/index.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-process-exit */
+
+const util = require('util');
+const path = require('path');
+const glob = util.promisify(require('glob'));
+const mv = require('move-file');
+const fs = require('fs').promises;
+const {existsSync} = require('fs');
+
+/**
+ * Move all markdown files and RSS feeds in src/site/content to the _exile
+ * directory.
+ * Optionally pass in an array of globs to be ignored.
+ * Items matching these globs will not be moved to _exile.
+ * @param {Array<string>} ignore An array of glob patterns to be ignored.
+ */
+async function isolate(ignore = []) {
```

Comment:
I'd run integrate or restore here before isolate. Otherwise, if you run isolate twice, e.g.:

```
npm run isolate -- src/site/content/en/fast/**/*
```

and sometime later:
```
npm run isolate -- src/site/content/en/accessible/**/*
```

you'd expect that you isolated 'accessible' dir, but in reality that dir is empty. So at least a restored() warning should be in place before re-running isolate command.

Reply:
ah good call. Instead of running integrate I think I may check if the `_exile` directory already exists and if it does then `isolate` will halt.